### ISSUE TITLE
[Update Request]: Kodi v21 (Omega)

### ISSUE THREAD

**Issue by rahaaatul** [microsoft/winget-pkgs#151476]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

XBMCFoundation.Kodi

### Package Version

20.5.0.0

### Please describe the changes you would like to see

There is a new release available `v21.0`.

Download 64 Bit
```
https://mirrors.kodi.tv/releases/windows/win64/kodi-21.0-Omega-x64.exe
```

Download 32 Bit
```
https://mirrors.kodi.tv/releases/windows/win32/kodi-21.0-Omega-x86.exe
```

**Comment by vikingnope** [comment]
https://github.com/microsoft/winget-pkgs/pull/151470

**Comment by rahaaatul** [comment]
> #151470

Is this is a coincidence? lol. 

Sorry I didn't see it. Shoud I close the issue or wait for the pull request to be merged?

**Comment by vikingnope** [comment]
Just wait for it to be merged then close it after :)

**Comment by Trenly** [comment]
* Resolved by #151470
